services/docker-compose.yml

```diff
@@ -208,6 +224,52 @@ services:
     networks:
       - default
 
+  traefik:
```

Comment:
well it reverse-proxies also to the webserver.
Actually I want to keep it like this as long as we do not have another one. I prefer knowing the application name than it's function since it's the only one that does it in there.

in deployment all the services are anyway prefixed with master, staging, production or whatever.